nifpga/bitfile.py

```diff
@@ -171,6 +181,61 @@ def __str__(self):
                 "\tOffset: %d\n" % self._offset)
 
 
+class Fxp_Register(Register):
+    """
+    A fixed point control or indicator from the front panel of the top level
+    FPGA VI.
+    """
+    def __init__(self, reg_xml, register):
+
+        self._copy_values_from_register(register)
+        datatype = reg_xml.find("Datatype")
+        fxp_xml = datatype.find("FXP")
+        self._signed = True if fxp_xml.find("Signed").text.lower() == 'true' else False
```

Comment:
In VHDL, I'd cut the wordiness and say something like:
self._signed = fxp_xml.find("Signed").text.lower() == 'true'

Is there a reason in Python for the apparently unneeded "if" stuff?

Reply:
I believe the reason we originally had the if else was for backwards compatibility. Right now this just matches the same behavior of the register parsing.